package.json

```diff
@@ -22,20 +22,17 @@
   "main": "dist/js-utils.cjs",
   "types": "dist/index.d.ts",
   "devDependencies": {
-    "@eslint/js": "^9.2.0",
-    "@types/node": "^20.12.7",
-    "@vitest/coverage-v8": "^1.5.2",
-    "eslint": "^9.2.0",
-    "globals": "^15.2.0",
-    "prettier": "^3.2.5",
-    "typescript": "^5.4.5",
-    "typescript-eslint": "^7.9.0",
-    "vite": "^5.2.10",
-    "vite-plugin-dts": "^3.9.0",
-    "vitest": "^1.5.2"
-  },
-  "dependencies": {
-    "jose": "^5.2.4"
+    "@eslint/js": "^9.9.1",
```

Comment:
Just for my own understanding - why have these been moved to dependencies?

Reply:
This change indicator is misleading, some dev dependancies were updated and I removed jose as it was not required.

Nothing else changed here.